yarn-project/Earthfile

```diff
@@ -47,6 +47,10 @@ build:
     ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
     RUN ./bootstrap.sh full
 
+workspace:
+    FROM +build
+    SAVE ARTIFACT /usr/src /usr/src
```

Comment:
Thoughts on doing this? I tried copying from +build but got a an error (can't remember what it said) so I made this new target that just exposed the fully built workspace

Reply:
This makes sense to me. Earthly just requires copies to be explicit like this. We can also put in the boxes earthfile if its the only one that uses.

Reply:
I did the same thing on my branch to build docs in earthly. I called the target `build-dev`